src/operator/elemwise_op_common.h

```diff
@@ -102,6 +102,48 @@ inline bool ElemwiseStorageType(const nnvm::NodeAttrs& attrs,
                                                  in_attrs, out_attrs);
 }
 
+template<bool cpu_only, bool rsp, bool csr>
+inline bool ElemwisePreferDenseStorageType(const nnvm::NodeAttrs& attrs,
```

Comment:
This function should be moved to binary_op instead of elemwise_op_common.h

Reply:
Done